apps/openchallenges/edam-etl/prepare-python.sh

```diff
@@ -12,4 +12,8 @@ eval "$(pyenv init -)"
 
 pyenv local $PYTHON_VERSION
 poetry env use $PYTHON_VERSION
-poetry install --with prod,dev
\ No newline at end of file
+poetry install --with prod,dev
+
+sudo apt-get update
+sudo apt-get install -y libmariadb-dev
+pip install packaging
```

Comment:
This package should be installed via Poetry.

Reply:
@tschaffter Thanks for catching that. I've removed the manual installation of MariaDB from prepare-python.sh